src/sgame/sg_main.cpp

```diff
@@ -2273,6 +2276,10 @@ can see the last frag.
 */
 void CheckExitRules()
 {
```

Comment:
In fact it very hard to figure out how it works because everything looks implicit on that part, even that array is undocumented, if there is a better way to do, why not, I just need to know how to do it.